bwapi/bwresources.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, bwproject):
             bwproject:  Brandwatch project.  This is a BWProject object.
         """
         self.project = bwproject
-        self.ids = {}
+        self.names = {}
```

Comment:
With a global search of .ids it looks like there are still some references to this attribute like self.tags.ids (where BWTags is child class of BWResource). They are found mostly in the _name_to_id methods. Can we confirm that this method still works with the new approach? Thank you!